packages/generator/package.json

```diff
@@ -22,7 +22,9 @@
     "build:clean": "run rimraf dist",
     "format": "run prettier --write '**/*.{ts,tsx,yaml,yml,json,md,mdx,js}'",
     "lint": "run eslint .",
-    "test": "node --experimental-vm-modules $(yarn bin jest)"
+    "test": "",
```

Comment:
This seems wrong.

Reply:
I agree this might be confusing, but the `test` script (or `target` called in NX world) depends on the other test targets (if present). See https://github.com/mittwald/api-client-js/pull/159/files#diff-15552e50b1b7c05b05b7ffe08ee47b5ed62b8d2039229c58972a42fd22a7381cR34

This is a way to call various "sub"-scripts via NX (`nx run-many -t test `)